### PR TITLE
Added new fonts to the website

### LINKED ISSUE #2: Change the font of the application
Bring in some nice new font

### PR DESCRIPTION
**Pull request by Nikhil7174**
## Description

Added new fonts to the website

## Issue Resolved

 Fixes #2.

## Checklist

Please make sure to review the following before submitting your PR:

- [x] I have read the [contribution guidelines](https://github.com/ken1000minus7/Bloggy/blob/master/README.md).
- [x] I have read the code of conduct.
- [x] I have reviewed my submission in detail.

### CODE REVIEW

File: src/frontend/src/App.js [hunk 1]
```diff
@@ -1,49 +1,47 @@
...
-import {Navbar} from "./components/Navbar";
-import {Route, Routes} from "react-router";
-import {NotFound} from "./pages/NotFound";
-import {HomePage} from "./pages/HomePage";
-import {BlogPage} from "./pages/BlogPage";
-import {LoginPage} from "./pages/LoginPage";
-import {ProfilePage} from "./pages/ProfilePage";
-import {CreatePage} from "./pages/CreatePage";
-import {SearchPage} from "./pages/SearchPage";
-import Footer from "./components/Footer";
```
Comment (ken1000minus7) [inline src/frontend/src/App.js]: We do not need these spacing changes

File: src/frontend/src/App.js [hunk 2]
```diff
@@ -1,49 +1,47 @@
...
     }
-    return (
-        <div className="App">
-            <Navbar changeTheme={changeTheme}/>
-            <Routes>
-                <Route index element={<HomePage />} />
-                <Route exact path="/home" element={<HomePage />} />
-                <Route exact path="/login" element={<LoginPage />} />
-                <Route exact path="/blog/:id" element={<BlogPage />} />
-                <Route exact path="/create" element={<CreatePage/>} />
```
Comment (ken1000minus7) [inline src/frontend/src/App.js]: Same here

File: src/frontend/src/components/Navbar.js [hunk 3]
```diff
@@ -1,114 +1,130 @@
...
-    useEffect(()=>{
-        const handleStorage = ()=>{
-            setUsername(localStorage.getItem("username"))
-            console.log(username)
-        }
-        window.addEventListener("storage",handleStorage)
-        return ()=>{
-            window.removeEventListener("storage",handleStorage)
-        }
-    },[])
```
Comment (ken1000minus7) [inline src/frontend/src/components/Navbar.js]: Same in this file

File: src/frontend/src/pages/HomePage.js [hunk 4]
```diff
@@ -46,16 +52,18 @@ export const HomePage = ()=>{
...
             </div>
             <div>
-                {
-                    latestBlogs && <BlogList blogList={latestBlogs.slice(0,Math.min(3,latestBlogs.length))} />
-                }
+                {latestBlogs && (
+                    <BlogList
+                        blogList={latestBlogs.slice(0, Math.min(3, latestBlogs.length))}
+                    />
+                )}
```
Comment (ken1000minus7) [inline src/frontend/src/pages/HomePage.js]: This change is unnecessary

File: src/frontend/tailwind.config.js [hunk 5]
```diff
@@ -1,19 +1,24 @@
...
-            'lg': {'max': '1023px'},
+            lg: { max: "1023px" },
 
-            'md': {'max': '767px'},
+            md: { max: "767px" },
 
-            'sm': {'max': '639px'},
-        }
+            sm: { max: "639px" },
+        },
```
Comment (ken1000minus7) [inline src/frontend/tailwind.config.js]: Same for this

**Review by ken1000minus7 — CHANGES_REQUESTED** [review]
Hey @Nikhil7174 , you do not need to make changes in the spacing and quotes, please avoid making changes to parts of code that don't need to be touched, I have commented on a few but almost all files need to be fixed

File: src/frontend/src/App.js (hunk 1 above)
Comment (Nikhil7174) [reply src/frontend/src/App.js]: I just saved the files in which i made the necessary changes with prettier and later with default document formatter to match the correct indentation. Still this problem is coming. But still will make indentation back to normal.

**Comment by Nikhil7174** [comment]
@ken1000minus7 I haven't touched any of these code, i just saved the files after adding new fonts to it using ctrl + s. Tried saving with both prettier and default document formatter. Please suggest me how to bring back the exact indentation back.

**Comment by ken1000minus7** [comment]
@Nikhil7174 Well you'll have to rollback the files to fix these changes, and you'll probably hv to change the settings of the prettier extension to disable auto formatting on saving, I hv never used prettier so I am not aware about the settings, you can look up how to change the settings

**Comment by Nikhil7174** [comment]
I did the best i could have done, can you please consider this.

**Comment by Nikhil7174** [comment]
My dark theme is also almost ready just waiting for this issue to be merged so that i can make a new pull request.

**Comment by ken1000minus7** [comment]
@Nikhil7174 The indentations are fixed, great 👌 , just send a screenshot of the page after these changes

**Comment by Nikhil7174** [comment]
Screenshot (114) - Copy
Screenshot (115) - Copy
Screenshot (116) - Copy
Screenshot (117) - Copy

**Review by ken1000minus7 — APPROVED** [review]
Lgtm

**Pull request merged**